script.js

```diff
@@ -1,4 +1,123 @@
+// game modes
+var gamemodeRollDice = 'game mode dice roll'
+var gamemodeChooseDice = 'game mode choose dice'
+var gamemodeChooseWinner = 'game mode choose winner'
+var gamemode = gamemodeRollDice
+oop
+
+// Player 
+var player1 = 'Player 1'
+var player2 = 'Player 2'
+var player = player1
+
+// Array for players' dice rolls
+var playerDice = []
+var player1Dice = []
+var player2Dice = []
+
+// Array for players' numbers
+var playerNumber = []
+var player1Number = []
+var player2Number = []
+
+var rollDice = function () {
+  // produces a decimal between 0 and 6
+
+  var randomDecimal = Math.random() * 6;
+
+  // take off the decimal
+  var randomInteger = Math.floor(randomDecimal);
+
+  // it's anumber from 0 - 5 ... add 1
+  var diceNumber = randomInteger + 1;
+
+  return diceNumber;
+};
+
+var getrandomDiceNumber = function () {
+  var randomDiceNumber = [rollDice(), rollDice()]
+  return randomDiceNumber;
+}
+
+var chooseWinner = function () {
+  if (player1Number > player2Number) {
+    return player1;
+  }
+  return player2;
+};
+
 var main = function (input) {
```

Comment:
Consider adding more comments/ pseudo code into main, so that it's easier to follow your line of thought :)